src/main/java/com/google/sps/data/Attendee.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+/** Class that represents an attendee. */
+public class Attendee {
+
+  private final String sessionId;
+  private final String screenName;
+  private Date timeLastPolled;
+  
+  /** Initializes an Attendee object
+   * @param {String} sessionId - the session the attendee is attached to.
+   * @param {String} screenName - the screen name of the attendee. 
+   * @param {Date} timeLastPolled - the time attendee was last polled.
+   */
+  public Attendee (String sessionId, String screenName, Date timeLastPolled) {
```

Comment:
Please create interfaces for all of these classes and any classes you all right, from now on. Something like:

class AttendeeInterface{
Blah();
}

class Attendee implements AttendeeInterface {
Blah() {
}
}